filters/builtin/builtin.go

```diff
@@ -165,6 +165,7 @@ func Filters() []filters.Spec {
 		NewOriginMarkerSpec(),
 		diag.NewRandom(),
 		diag.NewRepeat(),
+		diag.NewWrap(),
```

Comment:
NewContemtWrap maybe?

Reply:
For some reason we have `diag.NewRandom()` for `randomContent` and `diag.NewRepeat()` for `repeatContent`

Reply:
So either we keep the discrepancy or I can rename it